back-end/src/database/migrations/20211110000658-create-users.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING
+      },
+      email: {
+        type: Sequelize.STRING
+      },
+      password: {
+        type: Sequelize.STRING
+      },
+      role: {
+        type: Sequelize.STRING
```

Comment:
O [requisito 40](https://github.com/tryber/sd-010-a-project-delivery-app#40---desenvolva-a-tela-da-pessoa-administradora-de-forma-a-validar-o-formul%C3%A1rio-de-cadastro) do projeto sugere que a `role` tenha um valor _default_. Poderíamos colocar algo como `customer`? Estou levando em consideração que o projeto usa os termos `customer`, `seller` e `admin`.
```suggestion
        type: Sequelize.STRING,
        defaultValue: 'customer'
```

Reply:
Concordo, até vai evitar que tenhamos futuros problemas nesse projeto, valeu demais